src/main/java/org/opentestsystem/delivery/logging/EventParser.java

```diff
@@ -35,7 +36,7 @@
    */
   public Optional<EventInfo> parsePostHandle(final HttpServletRequest request, final HttpServletResponse response, final Object
     handler, EventLogger logger) {
-    return Optional.of(EventInfo.create(request.getPathInfo(), getEventDataFields(request)));
+    return Optional.of(EventInfo.create(StringUtils.trimToEmpty(request.getPathInfo()), getEventDataFields(request)));
```

Comment:
I know this isn't part of this change, but why are we using Optional here?  It seems that things are never optional?  If the call to EventInfo.create can somehow return a null EventInfo then this should be changed to Optional.fromNullable() otherwise this will throw an NPE.